examples/nidcpower_source_dc_voltage/measurement.py

```diff
@@ -63,96 +64,76 @@ def measure(
     """Source and measure a DC voltage with an NI SMU."""
     logging.info("Executing measurement: pin_names=%s voltage_level=%g", pin_names, voltage_level)
 
-    session_management_client = nims.session_management.Client(
-        grpc_channel=measurement_service.get_channel(
-            provided_interface=nims.session_management.GRPC_SERVICE_INTERFACE_NAME,
-            service_class=nims.session_management.GRPC_SERVICE_CLASS,
-        )
+    [session, reserved_session_info] = create_driver_session(
```

Comment:
Destructuring a tuple using list syntax seems weird to me.

Reply:
Updated.